packages/build/tests/plugins/validate/snapshots/tests.js.md

```diff
@@ -55,38 +58,41 @@ Generated by [AVA](https://ava.li).
     > Loading plugins␊
     ␊
     ┌─────────────────────────────┐␊
-    │     Netlify Build Error     │␊
+    │        Build failed         │␊
     └─────────────────────────────┘␊
     ␊
-    Error loading "/file/path" plugin:␊
-    Error: Invalid event 'onInvalid'.␊
-    Please use a valid event name. One of:␊
-     - onInit␊
-     - onPreGetCache␊
-     - onGetCache␊
-     - onPostGetCache␊
-     - onPreInstall␊
-     - onInstall␊
-     - onPostInstall␊
-     - onPreBuild␊
-     - onBuild␊
-     - onPostBuild␊
-     - onPreFunctionsBuild␊
-     - onFunctionsBuild␊
-     - onPostFunctionsBuild␊
-     - onPreFunctionsPackage␊
-     - onFunctionsPackage␊
-     - onPostFunctionsPackage␊
-     - onPreDeploy␊
-     - onPreSaveCache␊
-     - onSaveCache␊
-     - onPostSaveCache␊
-     - onSuccess␊
-     - onError␊
-     - onEnd␊
+    > Error message␊
+      Error: Invalid event 'onInvalid'.␊
+      Please use a valid event name. One of:␊
+       - onInit␊
```

Comment:
Looks like the list being printed out there is old. We need to show the smaller list of lifecycle hooks https://github.com/netlify/build#build-lifecycle